version.h.in

```diff
@@ -4,5 +4,7 @@
 #define PNET_VERSION_MAJOR @PNET_VERSION_MAJOR@
 #define PNET_VERSION_MINOR @PNET_VERSION_MINOR@
 #define PNET_VERSION_PATCH @PNET_VERSION_PATCH@
+#define PNET_VERSION_GIT "@PNET_VERSION_GIT@"
+#define PNET_VERSION_RTKERNEL_GIT "@PNET_VERSION_RTKERNEL_GIT@"
```

Comment:
PNET_VERSION_RTKERNEL_GIT is port-specific info and should not be in this file (also see os_kernel_revision above).

Reply:
Removed